tasks-reference/pam/text.xml

```diff
@@ -80,14 +80,14 @@ The statement is composed of 3 or 4 tokens:
     methods.  Some documentation states that we need to specify the full path of
     the module, but this creates problems because not all the systems install the
     modules in the same place: Linux-PAM on Gentoo is generally set up to load
-    them from <c>/lib/security</c>, but for example on AMD64 this become
+    them from <c>/lib/security</c>, but for example on AMD64 this becomes
     <c>/lib64/security</c>. The
     result is that providing the full path will lead to non-working <c>pamd</c>
-    files, and the right way to handle this is just states the module name <d /> the
+    files, and the right way to handle this is to just state the module name <d /> the
     PAM implementation will take care of finding the module.
   </li>
   <li>
-    The last token, which can consist of multiple items, describe the parameters
+    The last token, which can consist of multiple items, describes the parameters
```

Comment:
Long line. Please keep them below 80 characters.

Reply:
should fit now, thanks for spotting